sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/_configuration.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.0.6246, generator: {generator})
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any
+
+from azure.core.configuration import Configuration
+from azure.core.pipeline import policies
+
+from ._version import VERSION
+
+
+class FormRecognizerClientConfiguration(Configuration):
+    """Configuration for FormRecognizerClient.
+
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: azure.core.credentials.TokenCredential
+    :param endpoint: Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).
+    :type endpoint: str
+    """
+
+    def __init__(
+        self,
+        credential,  # type: "TokenCredential"
+        endpoint,  # type: str
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        if credential is None:
+            raise ValueError("Parameter 'credential' must not be None.")
+        if endpoint is None:
+            raise ValueError("Parameter 'endpoint' must not be None.")
+        super(FormRecognizerClientConfiguration, self).__init__(**kwargs)
+
+        self.credential = credential
+        self.endpoint = endpoint
+        self.credential_scopes = ['https://cognitiveservices.azure.com/.default']
+        kwargs.setdefault('sdk_moniker', 'azure-ai-formrecognizer/{}'.format(VERSION))
```

Comment:
Not the most pressing, but I would recommend regenerating with the latest autorest [release](https://github.com/Azure/autorest.python/releases/tag/v5.0.0-dev.20200326.1), we fixed the bug where we were including 'azure-' at the start of the UA name. According to telemetry [guidelines](https://github.com/Azure/azure-sdk/blob/6175e17f2b1077f2a2bcdb73bbc4e93bc5c0d8ad/docs/general/azurecore.md#telemetry-policy) it shouldn't be there

Reply:
Thanks! regenerated